### DISCUSSION TITLE
Allow users to sign up as "early access" or beta users

### DISCUSSION THREAD

**Discussion by s-taube** [freelawproject/foresight#73] [Ideas]
Stripe allows users to sign up for their "[early access program](https://insiders.stripe.dev/)" to "try the latest features and provide direct feedback to the teams that build them." I wonder if it might be useful to add something similar on courtlistener.
<img width="264" alt="early access" src="https://github.com/user-attachments/assets/fb05d67f-2470-42cd-b707-91b849c6c6ed">

**Comment by mlissner** [comment]
You mean like this: 

![image](https://github.com/user-attachments/assets/30935d8e-0bbc-4d1e-904e-a530979a264d)

:)

We don't use it much, but it does exist, at least in theory!
